src/powerquery-formatter/themes/scopeNameHelpers.ts

```diff
@@ -0,0 +1,173 @@
+        case PrimitiveTypeConstant.Binary:
+            return "constant.primitive-type.binary";
+        case PrimitiveTypeConstant.Date:
+            return "constant.primitive-type.date";
+        case PrimitiveTypeConstant.DateTime:
+            return "constant.primitive-type.date-time";
+        case PrimitiveTypeConstant.DateTimeZone:
+            return "constant.primitive-type.date-time-zone";
+        case PrimitiveTypeConstant.Duration:
+            return "constant.primitive-type.duration";
+        case PrimitiveTypeConstant.Function:
+            return "constant.primitive-type.function";
+        case PrimitiveTypeConstant.List:
+            return "constant.primitive-type.list";
+        case PrimitiveTypeConstant.Logical:
+            return "constant.primitive-type.logical";
+        case PrimitiveTypeConstant.None:
+            return "constant.primitive-type.none";
+        case PrimitiveTypeConstant.Null:
+            return "constant.primitive-type.null";
+        case PrimitiveTypeConstant.Number:
+            return "constant.primitive-type.number";
+        case PrimitiveTypeConstant.Record:
+            return "constant.primitive-type.record";
+        case PrimitiveTypeConstant.Table:
+            return "constant.primitive-type.table";
+        case PrimitiveTypeConstant.Text:
+            return "constant.primitive-type.text";
+        case PrimitiveTypeConstant.Time:
+            return "constant.primitive-type.time";
+        case PrimitiveTypeConstant.Type:
+            return "constant.primitive-type.type";
+        case RelationalOperator.LessThan:
+            return "constant.relational-operator.less-than";
+        case RelationalOperator.LessThanEqualTo:
+            return "constant.relational-operator.less-than-equal-to";
+        case RelationalOperator.GreaterThan:
+            return "constant.relational-operator.greater-than";
+        case RelationalOperator.GreaterThanEqualTo:
+            return "constant.relational-operator.greater-than-equal-to";
+        case UnaryOperator.Positive:
+            return "constant.unary-operator.positive";
+        case UnaryOperator.Negative:
+            return "constant.unary-operator.negative";
+        case UnaryOperator.Not:
+            return "constant.unary-operator.not";
+        case WrapperConstant.LeftBrace:
+            return "constant.wrapper.left-brace";
+        case WrapperConstant.LeftBracket:
+            return "constant.wrapper.left-bracket";
+        case WrapperConstant.LeftParenthesis:
+            return "constant.wrapper.left-parenthesis";
+        case WrapperConstant.RightBrace:
+            return "constant.wrapper.right-parenthesis";
+        case WrapperConstant.RightBracket:
+            return "constant.wrapper.right-bracket";
+        case WrapperConstant.RightParenthesis:
+            return "constant.wrapper.right-parenthesis";
+        default:
+            return "constant.unknown";
```

Comment:
Can this be typed in a way that allows us to do a `Assert.IsNever(constantKind)` as the default? That way if we had future constants (such as the upcoming "catch" keyword) we can discover errors during compile time.